rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -30,16 +33,19 @@ neutron_api_workers: 64
 neutron_rpc_conn_pool_size: "{{ rpc_conn_pool_size }}"
 neutron_rpc_response_timeout: "{{ rpc_response_timeout }}"
 neutron_rpc_thread_pool_size: "{{ rpc_thread_pool_size }}"
-neutron_db_max_overflow: 60
-neutron_db_max_pool_size: 120
-neutron_db_pool_timeout: 60
+neutron_db_max_overflow: "{{ db_max_overflow }}"
+neutron_db_max_pool_size: "{{ db_max_pool_size }}"
+neutron_db_pool_timeout: "{{ db_pool_timeout }}"
```

Comment:
I'd say either we should also change the keystone db variables here to use the global values, or we should do both the neutron ones and the keystone ones in their own respective pull requests. Here we have half and half.